adapters/nanointeractive/nanointeractive.go

```diff
@@ -0,0 +1,199 @@
+package nanointeractive
+
+import (
+	"encoding/json"
+	"fmt"
+	"github.com/mxmCherry/openrtb"
+	"github.com/prebid/prebid-server/adapters"
+	"github.com/prebid/prebid-server/errortypes"
+	"github.com/prebid/prebid-server/openrtb_ext"
+	"net/http"
+)
+
+type NanointeractiveAdapter struct {
+	endpoint string
+}
+
+// used for cookies and such
```

Comment:
Nitpick: I see you copied this comment from other adapters. :) Feel free to remove it if you wish, but it does no harm.

Reply:
busted :), I've removed the comment